dashboard/app/controllers/plc/enrollment_evaluations_controller.rb

```diff
@@ -0,0 +1,20 @@
+class Plc::EnrollmentEvaluationsController < ApplicationController
+  def perform_evaluation
+    authorize! :read, Plc::Course
+    @plc_user_course_enrollment = Plc::UserCourseEnrollment.find(params[:enrollment_id])
+    @questions = @plc_user_course_enrollment.plc_course.plc_evaluation_questions.all
```

Comment:
I'm not sure I understand here.

I actually need all the questions when I'm performing the evaluation - for the time being, we're showing them all at once. But even if we had multiple questions, wouldn't it be better to have only one query to the db?

I'm fairly certain I'm misunderstanding your comment here